ovos_m2v_pipeline/__init__.py

```
@@ -103,11 +104,15 @@ def handle_sync_intents(self, message: Message) -> None:
         timeout = self.config.get("timeout", 1)
-        self.intents = set(self._get_adapt_intents(timeout) + self._get_padatious_intents(timeout))
+        try:
+            self.intents = set(self._get_adapt_intents(timeout) + self._get_padatious_intents(timeout))
+            LOG.debug(f"Model2Vec registered intents: {len(self.intents)}")
+        except RuntimeError:
+            pass
         self._syncing = False
-        LOG.debug(f"Model2Vec registered intents: {len(self.intents)}")
 
-    def match_high(self, utterances: List[str], lang: str, message: Message) -> Optional[IntentHandlerMatch]:
+    @lru_cache()
```

Ruff (0.11.9)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)